example/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { Heading, Button } from 'nice2'; // eslint-disable-line import/no-unresolved, import/no-extraneous-dependencies
+import { Heading, Button } from 'oneui'; // eslint-disable-line import/no-unresolved, import/no-extraneous-dependencies
```

Comment:
Shouldn't it be `from '@textkernel/oneui';`

Reply:
Yes, you're right. I'll update it. Also, I noticed that the dist files are now capitalized (e.g. OneUI.js, OneUI.css) - we probably don't want that